apkid/rules/apk/packers.yara

```diff
@@ -1176,3 +1176,20 @@ rule manxi_sec : packer
   condition:
     is_apk and any of them
 }
+
+rule dexprotectx : packer
+{
+  meta:
+    description = "DexProtect X"
```

Comment:
Their name is also DexShellx. Would you also include it between parenthesis?

<img width="155" height="178" alt="Screenshot 2025-12-26 at 13 34 16" src="https://github.com/user-attachments/assets/b670344e-8055-4344-abeb-6adf8cf09156" />

`description = "DexProtect X (DexShellx)"`